style.css

```diff
@@ -889,96 +893,99 @@ input[id=email] {
   float: left;
   width: 50%;
   padding: 10px;
-
 }
 
-.about-column{
+.about-column {
   float: left;
   width: 50%;
   padding: 100px;
-
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
 }
 
 /* CARD IMAGES */
-.all_project-images{
+.all_project-images {
   height: 400px;
 }
 
 .proj-image1 {
-  background-image: url(./images/portfolioImage1.svg);
+  background-image: url(./images/eerste.svg);
   background-repeat: no-repeat;
   background-size: contain;
   background-position: center;
 }
 
 .proj-image2 {
-  background-image: url(./images/portfolioImage2.svg);
+  background-image: url(./images/portfolioImage3.svg);
   background-repeat: no-repeat;
   background-size: contain;
   background-position: center;
 }
 
 .proj-image3 {
-  background-image: url(./images/portfolioImage3.svg);
+  background-image: url(./images/portfolioImage4.svg);
   background-repeat: no-repeat;
   background-size: contain;
   background-position: center;
 }
 
 .proj-image4 {
-  background-image: url(./images/portfolioImage4.svg);
+  background-image: url(./images/portfolioImage1.svg);
   background-repeat: no-repeat;
   background-size: contain;
   background-position: center;
 }
 
-
 /* Clear floats after the columns */
-.row-example:after {
+.row-example::after {
   content: "";
   display: table;
   clear: both;
-
 }
+
 .row-example {
   border-radius: 24px;
   max-height: 700px;
-  margin:auto;
+  margin: auto;
   margin-top: 70px;
   background-color: white;
   max-width: 1170px;
-
+  margin-left: 20px;
+  margin-right: 20px;
 }
 
 /* Responsive layout - makes a two column-layout instead of four columns */
 @media screen and (max-width: 900px) {
-  .column-example, .about-column {
+  .column-example,
+  .about-column {
     width: 50%;
-    padding: 0px;
+    padding: 0;
   }
+
   .row-example {
-    margin:auto;
+    margin: auto;
     margin-top: 42px;
   }
-  
 }
+
 /* Responsive columns - one column layout (vertical) on small screens */
 @media screen and (max-width: 600px) {
-  .column-example, .about-column {
+  .column-example,
+  .about-column {
     width: 100%;
-    padding: 0px;
+    padding: 0;
   }
+
   .row-example {
-    margin:auto;
+    margin: auto;
     margin-top: 20px;
   }
-  .all_project-images{
+
+  .all_project-images {
```

Comment:
- [ ] As suggested by the previous reviewer, kindly ensure that your projects are in the center of the page like they are in the Figma design.

Reply:
Fixed.